models.py

```diff
@@ -1,6 +1,35 @@
 from room import db
+from flask_login import UserMixin
+from werkzeug.security import generate_password_hash, check_password_hash
+from room import login
+@login.user_loader
+def load_user(id):
+    return User.query.get(int(id))
 
+class ConciergeMii(db.Model):
+     mii_id = db.Column(db.Integer, primary_key=True, unique=True)
+     title = db.Column(db.String(6))
+     color1 = db.Column(db.String(6))
+     color2 = db.Column(db.String(6))
+     message1 = db.Column(db.String(100))
+     message2 = db.Column(db.String(100))
+     message3 = db.Column(db.String(100))
+     message4 = db.Column(db.String(100))
+     message5 = db.Column(db.String(100))
+     message6 = db.Column(db.String(100))
+     message7 = db.Column(db.String(100))
+     updated = db.Column(db.String(17))
+     movieid = db.Column(db.String(1))
```

Comment:
These should already be here, separated across several types:
https://github.com/WiiLink24/room-server/blob/84d90626cf878fcf6a9cb10c831cc7040814192d/models.py#L23-L48

I think your branch may've been slightly outdated, so we can work on updating that ^w^